packages/react-components/src/components/UserGuide/stories-helpers.tsx

```
@@ -0,0 +1,84 @@
+import './stories-helpers.css';
+
+export const AppContent = ({ onStartGuideClick }: { onStartGuideClick: () => void }): ReactElement => {
+  return (
+    <div className="main">
+      <div id="chat-list-column" className="column left">
+        <div className="column-header">
+          <Heading size="sm">Chats</Heading>
+        </div>
+        <div className="chats-list">
+          {[...Array(7)].map((_, index) => (
+            <div id={`chats-list-item-${index}`} className="chats-list-item">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)